scylla/src/transport/topology.rs

```diff
@@ -495,21 +498,31 @@ async fn query_peers(
         "system.local query response was not Rows",
     ))?;
 
-    let typed_peers_rows =
-        peers_rows.into_typed::<(IpAddr, Option<String>, Option<String>, Option<Vec<String>>)>();
+    let typed_peers_rows = peers_rows.into_typed::<(
+        Uuid,
```

Comment:
`host_id` is not a key column, so it is allowed to be null. While I don't expect it should be ever null in normal circumstances, we have at least one report which mentions that information in the peers table may be "glitched": https://github.com/scylladb/scylla-rust-driver/issues/235.

Let's be defensive and fetch it as `Option<Uuid>`, then filter out nodes for which the `host_id` is null.